Add C++ Queue and term entries (#751)
Codecademy/docs: content/cpp/concepts/queues/terms/size/size.md

---
Title: '.size()'

> Description: 'Returns the size of the queue.'

Reviewer: ```suggestion
Description: 'Returns the number of elements in the queue.'
```

Subjects:
  - 'Computer Science'
  - 'Game Development'
Tags:
  - 'Objects'
  - 'OOP'
  - 'Classes'
CatalogContent:
  - 'learn-c-plus-plus'
  - 'paths/computer-science'
---

> The `.size()` method returns the size of the queue.

Reviewer: ```suggestion
The `.size()` method returns the number of elements in the queue.
```

## Syntax

The `.size()` method is called on a queue using the following syntax:

```pseudo
queueName.size();
```

## Codebyte Example

The codebyte example below uses the `.size()` method on the `pizzas` queue:

```codebyte/cpp
#include <iostream>
#include <queue>

> int main(){

Reviewer 2: ```suggestion
int main() {
```
